src/common/msgtable.h

```diff
@@ -0,0 +1,2944 @@
+/**
+ * Messages from messages.conf
+ */
+enum msgtable_messages {
+	// 0 ~ 499: Messages of GM commands
+	// -----------------------
+	/** Warped. */
+	MSGTBL_WARPED = 0,
+	/** Map not found. */
+	MSGTBL_MAP_NOT_FOUND = 1,
+	/** Invalid coordinates, using random target cell. */
+	MSGTBL_INVALID_COORDINATES = 2,
+	/** Character not found. */
+	MSGTBL_CHARACTER_NOT_FOUND = 3,
+	/** Jump to %s */
+	MSGTBL_JUMP_TO_NAME = 4,
+	/** Jump to %d %d */
+	MSGTBL_JUMP_TO_XY = 5,
+	/** Your save point has been changed. */
+	MSGTBL_SAVE_POINT_CHANGED = 6,
+	/** Warping to save point. */
+	MSGTBL_WARP_TO_SAVE_POINT = 7,
+	/** Speed changed. */
+	MSGTBL_SPEED_CHANGED = 8,
+	/** Options changed. */
+	MSGTBL_OPTIONS_CHANGED = 9,
+	/** Invisible: Off */
+	MSGTBL_INVISIBLE_OFF = 10,
+	/** Invisible: On */
+	MSGTBL_INVISIBLE_ON = 11,
+	/** Your job has been changed. */
+	MSGTBL_JOB_CHANGED = 12,
+	/** You've died. */
+	MSGTBL_YOU_DIED = 13,
+	/** Character killed. */
+	MSGTBL_CHARACTER_KILLED = 14,
+	/** Unknown */
+	MSGTBL_UNKNOWN = 15,
+	/** You've been revived! */
+	MSGTBL_REVIVED = 16,
+	/** HP, SP recovered. */
+	MSGTBL_HP_SP_RECOVERED = 17,
+	/** Item created. */
+	MSGTBL_ITEM_CREATED = 18,
+	/** Invalid item ID or name. */
+	MSGTBL_INVALID_ITEMID_NAME = 19,
+	/** All of your items have been removed. */
+	MSGTBL_ITEMS_REMOVED = 20,
+	/** Base level raised. */
+	MSGTBL_BASE_LEVEL_RAISED = 21,
+	/** Base level lowered. */
+	MSGTBL_BASE_LEVEL_LOWERED = 22,
+	/** Job level can't go any higher. */
+	MSGTBL_CANT_INCREASE_JOB_LEVEL = 23,
+	/** Job level raised. */
+	MSGTBL_JOB_LEVEL_RAISED = 24,
+	/** Job level lowered. */
+	MSGTBL_JOB_LEVEL_LOWERED = 25,
+	/** [%d] seconds left until you can use */
+	MSGTBL_SECONDS_UNTIL_USE = 26,
+	/** Storage has been not loaded yet. */
+	MSGTBL_STORAGE_NOT_LOADED = 27,
+	/** No player found. */
+	MSGTBL_NO_PLAYER_FOUND = 28,
+	/** 1 player found. */
+	MSGTBL_1_PLAYER_FOUND = 29,
+	/** %d players found. */
+	MSGTBL_N_pLAYERS_FOUND = 30,
```

Comment:
Minor nitpick: a lowercase `p` slipped in

Reply:
Ooops, I will fix now. nice catch :D